litmus-portal/frontend/src/models/redux/workflow.ts

```diff
@@ -54,6 +54,7 @@ export interface WorkflowData {
 export interface WorkflowManifest {
   engineYAML: string;
   manifest: string;
+  weights: experimentMap[];
```

Comment:
I think we can do weight updation without redux as well at the state level. We should move with that approach.